<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuViews">
    <property name="title">
     <string>Views</string>
    </property>
    <addaction name="actionThomas"/>
    <addaction name="actionloves"/>
    <addaction name="actionJean"/>
   </widget>
   <widget class="QMenu" name="menuMore">
    <property name="title">
     <string>More</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuViews"/>
   <addaction name="menuMore"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionThomas">
   <property name="text">
    <string>Thomas</string>
   </property>
  </action>
  <action name="actionloves">
   <property name="text">
    <string>loves</string>
   </property>
  </action>
  <action name="actionJean">
   <property name="text">
    <string>Jean</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
